6/27 Content Volume (#75)
Contenda-Team/Contenda-landing-page: src/posts/contentvolume.md

---
layout: "../layouts/BlogPost.astro"
title:
    "How much weekly content volume should I aim to produce?"
slug: content-volume
description:
    "More weekly content volume is positively correlated with more traffic, but how can my team keep up?"
author: "Lilly Chen"
added: "June 27 2023"
tags: ["opinion"]
---

Karl Hughes of Draft.dev posted about how developer tool companies who post more content per week, are coorelated with [more monthly visitors](https://www.linkedin.com/posts/karllhughes_ive-never-heard-a-developer-tool-company-activity-7079186734550323200-Yx7S?utm_source=share&utm_medium=member_desktop).

> ![Karl Hughes's post on content volume](/blogimages/contentvolume.jpg)

Reviewer: can you add a caption or something maybe? its not obvious its an image and my first instinct was to try to click on the company hyperlinks

## How good does every piece of content need to be?

There are two rules for content performance:

- Never put out content you think is bad
- Good content does not have to be long

If you think a piece is bad, don't publish it. It's okay to be uncertain or think that it's not the absolute best thing you've ever made. Those are normal feelings. But if you believe in your heart that something you made is bad, then don't put your name on it. 

The second corollary is that good content does not have to be long. A good example of this is tweet threads. There are plenty of super informative tweet threads that don't exceed 280 characters a tweet. 

## How much content should I produce?

There's lots of factors that influence this decision, including your size of team, stage of product, and market landscape. It is typically more important to be *consistent* about volume, rather than doing it in spurts. Here's some quick tips:

> > If you ever produce content in bulk, don't publish it all at once! Schedule them out weekly. 

Reviewer: ```suggestion
- If you ever produce content in bulk, don't publish it all at once! Schedule them out weekly. 
```

currently the way it outputs is just one nested quote
![Screen Shot 2023-06-27 at 8 36 50 AM](https://github.com/Contenda-Team/Contenda-landing-page/assets/8921721/da748e11-858d-4671-b76e-66dad482f586)

> > *Adherence* to a content calendar is very important. Make routines that you can keep without exhausting yourself. 

Reviewer: ```suggestion
- *Adherence* to a content calendar is very important. Make routines that you can keep without exhausting yourself. 
```

> > Have fun! You are sharing your knowledge (and hopefully your passion and joy!) with the world. It is a beautiful privelege. 

Reviewer: ```suggestion
- Have fun! You are sharing your knowledge (and hopefully your passion and joy!) with the world. It is a beautiful privilege. 
```

## Help! I think I overcommitted to a content volume that I can't maintain

Don't give it completely! Just dial it back. If you were doing 2-3 pieces a week, but feel yourself burning out, don't give it up completely. Spread it out to 1 piece a week. If you're working with a team, get your team to contribute. Most people are happy to guest write or collaborate on a content piece, especially if it's not something they do super often. If you're struggling still, consider repurposing content instead of creating net new pieces everytime. 

> Repurposing content can include finding something you've already produced and transforming it into a new medium suchas Blog --> Twitter Thread or Video --> Blog. It can also be an aggregation of existing content into a listicle like "Top 10 Pieces of Advice for New Grads". Something we do a lot at Contenda is sequel posts like "Feature X, Part 2!", especially on posts where the original topic seemed to be doing well. 

Reviewer: ```suggestion
Repurposing content can include finding something you've already produced and transforming it into a new medium such as Blog --> Twitter Thread or Video --> Blog. It can also be an aggregation of existing content into a listicle like "Top 10 Pieces of Advice for New Grads". Something we do a lot at Contenda is sequel posts like "Feature X, Part 2!", especially on posts where the original topic seemed to be doing well. 
```

> A last option if you're struggling to find help or keep up the pace, is to hire a technical content agency. Karl, who inspired this blog, runs a great team at Draft.dev. If you need to take a break, but want to keep your content volume, give them a call! 

Reviewer: i'm fine with leaving `draft.dev` in if they are gonna comarket but seems a bit last minute to do that?
